etc/sysconfig/ceph

```diff
@@ -24,3 +24,11 @@ TCMALLOC_MAX_TOTAL_THREAD_CACHE_BYTES=134217728
 # currently active on the node.
 #
 CEPH_AUTO_RESTART_ON_UPGRADE=no
+
+## unmap all devices, or only those configured in /etc/ceph/rbdmap
+#
+# By default, "rbdmap unmap" (systemctl stop rbdmap) unmounts and unmaps
+# all locally mapped RBD images. Configuring "no" here overrides this
+# behaviour so that only RBD images specified in /etc/ceph/rbdmap are
+# unmounted and unmapped.
+RBDMAP_UNMAP_ALL=yes
```

Comment:
@dillaman thanks for the feedback.

I didn't try to factor anything out in this round, as I was (and still am) unsure whether the existing unmap behaviour should be kept, or completely dropped. My preference would probably be to just drop it, unless someone thinks that there are users relying on the unmap-all-by-default behaviour.

If you think it's worth keeping the existing behaviour, then I'm happy